integration/utils/follower_test.go

```diff
@@ -28,6 +28,24 @@ func TestTxFollower(t *testing.T) {
 	f.Stop()
 }
 
+// TestTxFollowerFollowAfterStop creates new follower with a fixed block height and stops it.
+func TestTxFollowerFollowAfterStop(t *testing.T) {
+	// TODO(rbtz): test against a mock client, but for now we just expire
+	// the context so that the followere wont be able to progress.
+	ctx, cancel := context.WithCancel(context.Background())
+	cancel()
```

Comment:
this cancels the context right away, which i assume is on purpose, does moving this to line 44 just before the stop make the test more representative? or not really?

Reply:
sadly for now there is no mock available for the access.Client interface.  It is in my TODO to generate one.  Once it is available this hack can be removed.